### PR TITLE
Fixed kaggle scripts #65-68

### PR DESCRIPTION
**Pull request by Kadam-Tushar**
Modified scripts from #65 #66 #67 #68 

- Fixed some typos like adding `kaggle` and `data_source` field in every kaggle scripts
- Fixed some urls of some datasets to `Author/dataset_name` (kaggle api format)

 @henrykironde  As advised in #88 I have made separate PR by adding  `kaggle dataset` as a keyword and committed the changes as an original author.
 
 Note: As soon as you will merge this PR, I will send 2nd PR as another author for the remaining scripts to avoid any conflicts.

### CODE REVIEW

**Comment by henrykironde** [comment]
I have fixed the build problem for the moment. Docker limits the number of pulls we make. We shall solve resolve that later.
Could you test these scripts and paste some sample output.Make sure the data is in the database. 

Sample example using Iris
```
(deepforest) ➜  retriever-recipes git:(dad0f5a) ✗ retriever install postgres iris
=> Installing iris
Creating database iris...
Installing into existing database
Downloading bezdekIris.data: 3.00B [00:00, 8.26B/s]                                                                                    
Installing iris.Iris
Progress: 100%|██████████████████████████████████████████████████████████████████████████████████| 151/151 [00:00<00:00, 1556.83rows/s]
Done!
```
Screen Shot 2021-02-01 at 9 30 58 AM

**Comment by Kadam-Tushar** [comment]
@henrykironde  Sure I will do that, One quick question: do I have to show output from Postgres only  or any option will work like `retriever install csv iris`?

**Comment by Kadam-Tushar** [comment]
@henrykironde 
There were some problems while installing `coronavirus-south-korea` and `foreign-exchange-rates-2000-2019` datasets in postgres/mysql because of following problems in script:
- `size` field in given scripts should be only attached with char datatypes (it was attached with [doubles](https://github.com/weecology/retriever-recipes/pull/65/files#diff-3a74cc551ca8df7c06036fe0943cd10a8328e46299ce989c92d0bef5c195b06aR108) and ints too which was causing compilation problems in create table statements ). Hence removed those `size` key-value pairs in scripts from doubles and ints.
- for `coronavirus-south-korea` script `int` was overflowing in some columns hence replaced it with `bigint`
-  `index` column was missing in `foreign-exchange-rates-2000-2019` and fixed datatype of `timserie` column.

**Comment by Kadam-Tushar** [comment]
Adding output images for reference 

1.  coronavirus datasets

coronavirus-cli-2
coronavirus_cli
coronavirus-south-korea_pg

2. new-york air bnb dataset

ab_nyc
new_york_open_data

3. Credit card dataset

credit_card
credit_card_post

4. Foreign Exchange dataset

foreign_exchange_pg
forign_exchange_cli

File: scripts/coronavirus_south_korea.json [hunk 1]
```diff
@@ -0,0 +1,541 @@
...
+                    }
+                ]
+            },
+            "url": "kimjihoo/coronavirusdataset"
+        }
+    ],
+    "kaggle": "True",
+    "data_source": "dataset",
+    "retriever_minimum_version": "2.1.0",
+    "title": "coronavirus-south-korea",
```
Comment (henrykironde) [inline scripts/coronavirus_south_korea.json]: DS4C: Data Science for COVID-19 in South Korea

File: scripts/new_york_city_airbnb_open_data.json [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                ]
+            },
+            "url": "dgomonov/new-york-city-airbnb-open-data"
+        }
+    ],
+    "kaggle": "True",
+    "data_source": "dataset",
+    "retriever": "True",
+    "retriever_minimum_version": "2.1.0",
+    "title": "new-york-city-airbnb-open-data",
```
Comment (henrykironde) [inline scripts/new_york_city_airbnb_open_data.json]: New York City Airbnb Open Data

File: scripts/foreign_exchange_rates_2000_2019.json [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+                ]
+            },
+            "url": "brunotly/foreign-exchange-rates-per-dollar-20002019"
+        }
+    ],
+    "kaggle": "True",
+    "data_source": "dataset",
+    "retriever": "True",
+    "retriever_minimum_version": "2.1.0",
+    "title": "foreign-exchange-rates-2000-2019",
```
Comment (henrykironde) [inline scripts/foreign_exchange_rates_2000_2019.json]: Foreign Exchange rates for 2000 t0 2019

File: scripts/credit_card_fraud.json [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+                ]
+            },
+            "url": "mlg-ulb/creditcardfraud"
+        }
+    ],
+    "kaggle": "True",
+    "data_source": "dataset",
+    "retriever": "True",
+    "retriever_minimum_version": "2.1.0",
+    "title": "credit-card-fraud",
```
Comment (henrykironde) [inline scripts/credit_card_fraud.json]: Credit Card Fraud Detection Dataset

File: scripts/coronavirus_south_korea.json [hunk 5]
```diff
@@ -0,0 +1,541 @@
+{
+    "archived": "zip",
+    "citation": "",
+    "description": "Dataset of COVID-19 in South Korea",
```
Comment (henrykironde) [inline scripts/coronavirus_south_korea.json]: NeurIPS 2020 Data Science for COVID-19,  South Korea

**Review by henrykironde — COMMENTED** [review]
@Kadam-Tushar, I have made some changes and created a PR on your fork. 
1) Merge the PR
2) Revise as suggested and run the commands again to make sure that all scripts are working well.

**Comment by Kadam-Tushar** [comment]
@henrykironde  do I have to make this new commit (For suggestions) as a different --author ?
Edit : I have done commit, let me know if any changes are needed. 
PS: I have tested scripts after changes and they are working fine.

**Comment by henrykironde** [comment]
Thanks for the work @Kadam-Tushar.

**Pull request merged**